README.rst

```diff
@@ -87,155 +129,156 @@ must be present and contain a dict with Zyte API parameters:
-            )
+-   ``Request.body`` becomes ``httpRequestBody`` even if ``httpResponseBody``
+    is unset.
 
-        def parse(self, response):
-            yield {"URL": response.url, "HTML": response.body}
+-   You can set ``httpResponseBody`` to ``False`` (which unsets the parameter),
+    and not set ``browserHtml`` or ``screenshot`` to ``True``. In this case,
+    ``Request.headers`` is mapped as ``requestHeaders``.
 
-            print(response.raw_api_response)
-            # {
-            #     'url': 'https://quotes.toscrape.com/',
-            #     'statusCode': 200,
-            #     'browserHtml': '<html> ... </html>',
-            # }
+-   You can set ``httpResponseBody`` to ``True`` and also set ``browserHtml``
+    or ``screenshot`` to ``True``. In this case, ``Request.headers`` is mapped
+    both as ``customHttpRequestHeaders`` and as ``requestHeaders``, and
+    ``browserHtml`` is used as the Scrapy response body.
+
+
+Using transparent mode
+----------------------
+
+Set the ``ZYTE_API_TRANSPARENT_MODE`` setting to ``True`` to handle Scrapy
+requests as follows:
+
+-   Requests with the ``zyte_api_automap`` request meta key set to ``False``
+    are *not* sent through Zyte API.
+
+-   Requests with the ``zyte_api`` request meta key set to a ``dict`` are sent
+    through Zyte API with manually-defined parameters. See **Sending requests
+    with manually-defined parameters** above.
+
+-   All other requests are sent through Zyte API with automatically-mapped
+    parameters. See **Sending requests with automatically-mapped parameters**
+    above.
+
+    You do not need to set the ``zyte-api-automap`` request meta key to
+    ``True``, but you can set it to a dictionary to extend your request
+    parameters.
+
+
+Setting default parameters
+==========================
+
+Often the same configuration needs to be used for all Zyte API requests. For
+example, all requests may need to set the same geolocation, or the spider only
+uses ``browserHtml`` requests.
+
+The following settings allow you to define Zyte API parameters to be included
+in all requests:
+
+-   ``ZYTE_API_DEFAULT_PARAMS`` is a ``dict`` of parameters to be combined with
+    manually-defined parameters. See **Sending requests with manually-defined
+    parameters** above.
+
+    You may set the ``zyte_api`` request meta key to an empty ``dict`` to only
+    use default parameters for that request.
+
+-   ``ZYTE_API_AUTOMAP_PARAMS`` is a ``dict`` of parameters to be combined with
+    automatically-mapped parameters. See **Sending requests with
+    automatically-mapped parameters** above.
+
+For example, if you set ``ZYTE_API_DEFAULT_PARAMS`` to
+``{"geolocation": "US"}`` and ``zyte_api`` to ``{"browserHtml": True}``,
+``{"url: "…", "geolocation": "US", "browserHtml": True}`` is sent to Zyte API.
+
+Parameters in these settings are merged with request-specific parameters, with
+request-specific parameters taking precedence.
```

Comment:
It'd be nice to clarify behavior of these options for the requests which are not marked as manual or auto-mapped zyte API requests.